scripts/deploy.sh

```
@@ -0,0 +1,215 @@
+            --query 'DBInstances[0].[DBInstanceStatus,AllocatedStorage,DBInstanceClass]' \
+            --output table
+        
+        # Check S3 bucket size
+        echo -e "\nS3 Storage:"
+        aws s3 ls s3://ch11-bankruptcy-documents --recursive --summarize | tail -n 2
+        
+        # Check SQS queues
+        echo -e "\nSQS Queue Depths:"
+        for queue in petition-processing backfill-processing; do
+            DEPTH=$(aws sqs get-queue-attributes \
+                --queue-url https://sqs.$AWS_REGION.amazonaws.com/$(aws sts get-caller-identity --query Account --output text)/ch11-$queue \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)